File: test/export/vb/06-segun.vb
Module SIN_TITULO
	Sub Main()
		Dim c As Double
		c = Double.Parse(Console.ReadLine())
		Select Case c
		Case 1
			Console.WriteLine("1")
		Case 3, 5, 9
			Console.WriteLine("3, 5 o 9")
		Case 7
			Console.WriteLine("7")
		Case Else
			Console.WriteLine("ni 1, ni 3, ni 5, ni 7, ni 9")
		End Select
	End Sub
End Module

File: test/export/cs/06-segun.cs
using System;
namespace PSeInt {
	class sin_titulo {
		static void Main(string[] args) {
			int c;
			c = int.Parse(Console.ReadLine());
			switch (c) {
			case 1:
				Console.WriteLine("1");
				break;
			case 3: case 5: case 9:
				Console.WriteLine("3, 5 o 9");
				break;
			case 7:
				Console.WriteLine("7");
				break;
			default:
				Console.WriteLine("ni 1, ni 3, ni 5, ni 7, ni 9");
			}
		}
	}
}
